raiden-dapp/src/components/account/Withdrawal.vue

```diff
@@ -156,15 +154,24 @@ export default class Withdrawal extends Mixins(BlockieMixin) {
   withdraw: Token | null = null;
 
   async mounted() {
-    this.balances = await this.$raiden.getRaidenAccountBalances();
-
-    if ((this.udcToken.balance as BigNumber).gt(Zero)) {
-      this.balances = uniqBy(
-        [...this.balances].concat(this.udcToken),
-        (token) => token.address
-      );
-    }
+    const balancesWithoutMainnetUtilityToken = await this.$raiden.getRaidenAccountBalances();
```

Comment:
These var names are not good: you can't be sure mainnet SVT won't come on this list. It just doesn't right now, but could if TN deployment is enabled there.